src/components/VInput.vue

```diff
@@ -1,13 +1,16 @@
 <template>
   <div>
     <label class="flex flex-col text-secondary-text">
-      <span class="text-left mb-1 text-sm">{{ label }}</span>
+      <span class="text-left mb-1 text-sm" :class="isFocused ? 'text-primary-green' : ''">
```

Comment:
We can use class binding here
```suggestion
      <span class="text-left mb-1 text-sm" :class="{ 'text-primary-green' : isFocused }">
```